src/app.js

```diff
@@ -1,55 +1,159 @@
-const bodyParser = require('body-parser');
-const express = require('express');
-const GitHubApi = require('github');
-const nacl = require('tweetnacl');
-nacl.util = require('tweetnacl-util');
-
-const username = 'yourusername';  // TODO: your GitHub username here
-const github = new GitHubApi({ debug: true });
+const bodyParser = require("body-parser");
+const express = require("express");
+const GitHubApi = require("github");
+const nacl = require("tweetnacl");
+nacl.util = require("tweetnacl-util");
+
+require("dotenv").config();
+
+const username = "Lambdarines"; // TODO: your GitHub username here
+const github = new GitHubApi({
+  debug: true
+});
 const server = express();
+const token = process.env.GITHUB_TOKEN;
+let client_id = "";
+
+console.log('token', token);
+
+let boxKey, nonce;
+server.use(bodyParser.json());
 
 // Generate an access token: https://github.com/settings/tokens
 // Set it to be able to create gists
+server.use(bodyParser.json());
 github.authenticate({
-  type: 'oauth',
-  token: process.env.GITHUB_TOKEN
+  type: "oauth",
+  token
 });
 
 // Set up the encryption - use process.env.SECRET_KEY if it exists
 // TODO either use or generate a new 32 byte key
+// const secretKey = Uint8Array.from(process.env.SECRET_KEY);
+// const nonce = process.env.NONCE;
 
-server.get('/', (req, res) => {
+server.get("/", (req, res) => {
   // TODO Return a response that documents the other routes/operations available
+  github.users.getForUser({
+    username
+  }).then(response => {
+    res.json(response.data);
+    console.log("client id", response.data.id);
+    client_id = response.data.id;
+    // console.log(response.data);
```

Comment:
get rid of this is you don't need it

Reply:
Another option is to have debug logging be controlled by some sort of `debug = True / False` variable - but agreed that if it's purely exploratory logging that isn't useful ongoing, it should be removed.